deploy/crds/operators.coreos.com_servicebindings_crd.yaml

```diff
@@ -259,6 +260,8 @@ spec:
                 type: object
               type: array
           type: object
+          required:
+              - "services"
```

Comment:
Oh I thought we had removed "service" 🤔

Reply:
yes, but if I'm not wrong Kuberners allows us to make CR with singular and plural fields and we're not dealing with service so need to add validation to use services.